File: codeforces/Codeforces-Round-881-Div3/pA.cpp
#include <bits/stdc++.h>

#define int long long

using namespace std;

signed main(void)
{
	ios_base::sync_with_stdio(false);
	cin.tie(0);
	
	int q;
	cin >> q;
	while(q--)
	{
		int n;
		cin >> n;
		deque<int> dq;
		for(int i=0;i<n;i++)
		{
			int input;
			cin >> input;
			dq.push_back(input);
		}
		
		sort(dq.begin(),dq.end());
		
		int ans = 0;
		
		while( dq.size() > 1 )
		{
			ans += dq.back() - dq.front();
			dq.pop_front();
			dq.pop_back();
		}
		
		cout << ans << "\n";
	}
	
	return 0;
} 

File: ITSA/2023-12-2/pA.py
n = int(input())

num = 1
for i in range(n):
    u = int(input())
    num *= u

ans = 0
while( num != 0 ):
    if( num % 10 == 0 and num >= 10 ):
        ans += 1
    num //= 10

print(ans)